desktop/release-notes.md

```diff
@@ -33,15 +33,71 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > [Mac with Apple
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
-> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.0-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
 > .button .primary-btn }
-> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.0-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
 > .button .primary-btn }
-> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
 > .button .primary-btn }
 
 ### New
 
+- Added a footer to the Docker Dashboard with general information about the Docker Desktop update status and Docker Engine statistics
+- Re-designed the containers table, adding:
+  - A button to copy a container ID to the clipboard
+  - A pause button for each container
+  - Column resizing for the containers table
+  - Persistence of sorting and resizing for the containers table
+  - Bulk deletion for the containers table
+
+### Upgrades
+
+- [Compose v2.6.0](https://github.com/docker/compose/releases/tag/v2.6.0)
+- [Docker Engine v20.10.16](https://docs.docker.com/engine/release-notes/#201016)
+- [containerd v1.6.4](https://github.com/containerd/containerd/releases/tag/v1.6.4)
+- [runc v1.1.1](https://github.com/opencontainers/runc/releases/tag/v1.1.1)
+- [Go 1.18.2](https://github.com/golang/go/releases/tag/go1.18.2)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed Docker Desktop hanging when quitting while paused
+- Fixed the Kubernetes cluster not resetting properly after the PKI expires
+- Fixed the extensions' marketplace not using the defined http proxies
+- Improved the Dashboard logs search to allow spaces
+- Fixed middle button click on links in the Dashboard to correctly open external links
+- Fixed middle button click on buttons on the Dashboard to behave as a left button click instead of opening a blank window
+
+#### For Mac
+
+- Avoid creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list
```

Comment:
```suggestion
- Fixed an issues which avoids creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list
```

Reply:
Changed to 
> Fixed an issue to avoid creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list.

as it’s the other way around: the issue was that it created these folders, and we don’t want that.